### ISSUE TITLE
DiffPose will provide  Anterior-posterior (AP) and lateral view  2D/3D registration method?

### ISSUE THREAD

**Issue by BigPandaCPU** [eigenvivek/DiffPose#39]
Hi Vivek,

Thank you for your work. 
I have test the DiffDRR, the registrations errors below!  AP view registraion error
![05](https://github.com/user-attachments/assets/809034e5-b4cb-4cac-b2fe-27894a976671)

The erros of the dy is 4.49, because we only use the AP view to registration.  

I have extend the DiffDRR registration to create AP view DRR and Lateral DRR.

![06](https://github.com/user-attachments/assets/873142bf-5c10-4281-bc62-e66b5c6976a5)

The registations errors below!

![07](https://github.com/user-attachments/assets/8f05f6df-56b1-4f89-84a9-0a23ef9c7026)

That the errors of dy is 5.29mm.

**Comment by eigenvivek** [comment]
Hi @BigPandaCPU , not sure where your issues is arising from. In general, failure of registration to converge is either due to 
- A poor initial pose estimate
- Bad hyperparameters (especially too high learning rate for the optimizers)

Specifically for your misregistration, it doesn't look like the `by` is misestimated. Rather, I would expect `gamma` to be off, since the moving DRR is rotated relative to the fixed DRR.

Feel free to open a new issue (probably in the DiffDRR repo) if your issue persists.
